paymentsheet/src/androidTest/java/com/stripe/android/paymentsheet/CustomerSheetPage.kt

```diff
@@ -66,6 +67,28 @@ internal class CustomerSheetPage(
         clickPrimaryButton(CUSTOMER_SHEET_CONFIRM_BUTTON_TEST_TAG)
     }
 
+    fun clickEditButton() {
+        val editButtonMatcher = hasText("EDIT")
```

Comment:
for these matchers, should we add a test tag to the buttons instead of using the `hasText` matchers? I suspect that if we changed this text, the tests that depend on this might fail in somewhat hard-to-debug ways